ClientHints.php

```diff
@@ -236,7 +236,7 @@ public static function factory(array $headers): ClientHints
         $fullVersionList = [];
 
         foreach ($headers as $name => $value) {
-            switch (\str_replace('_', '-', \strtolower($name))) {
+            switch (\str_replace('_', '-', \strtolower(\strval($name)))) {
```

Comment:
I would replace \strval($name) with (string)$name
> [EA] '(string) $language' can be used instead (reduces cognitive load, up to 6x times faster in PHP 5.x). 
 Inspection info: Analyzes if PHP4 functions (intval, floatval, doubleval, strval) are used for type casting and generates hints to use PHP5's type casting construction (i.e. '(type) parameter').

Reply:
Not sure if that still makes a difference in PHP 7 or 8. But will change it.